src/integration.test.ts

```diff
@@ -0,0 +1,22 @@
+import { SkynetClient } from "./client";
+// import { Buffer } from "buffer";
+import { pki } from "node-forge";
+// import { readData } from "./utils";
+
+const { publicKey, privateKey } = pki.ed25519.generateKeyPair();
+const client = new SkynetClient("https://siasky.dev");
+
+const datakey = "HelloWorld";
+const json = { data: "thisistext" };
+
+// skip - used to verify end-to-end flow
+describe.skip("siasky.dev end to end", () => {
+  it("to update the file in the SkyDB", async () => {
```

Comment:
could improve on these descriptions

`describe.skip("skydb integration test")`
`it("should be able to both `setJSON` and `getJSON`")`